### PR TITLE
Change imports (and other) from ubuntu-phonedations to ubports

### PR DESCRIPTION
**Pull request by jezek**
Follows PRs #6 & #7 

This PR proposes to cut ties from ubuntu-phonedations repo for good. This step is essential for further development of ```nuntium```. For example without this, any change to [nuntium/ofono package](https://github.com/ubports/nuntium/tree/xenial/ofono) wouldn't have any effect, cause ```nuntium``` [builds against ubutu-ponedations packages](https://github.com/ubports/nuntium/blob/38940464d532a145412c47d8901c1a74fb662eff/cmd/nuntium/main.go#L29-L30). I've splitted this changes into some commits for better clarity, with some insights in commit descriptions.

### CODE REVIEW

**Comment by lduboeuf** [comment]
tested on BQ4.5, no regressions found and also the MMSEnabled toggle is working now. ( the MMS is not retrieved as expected when set to off , before it was not the case )

**Comment by lduboeuf** [comment]
I would split this PR in two, one with the cherry-pick from phondation and the other for namespace migration ( just say that but i'm not the maintainer )

**Comment by jezek** [comment]
@lduboeuf 
> I would split this PR in two, one with the cherry-pick from phondation and the other for namespace migration ( just say that but i'm not the maintainer )

Yep, it would be more logical. Maybe it's also fine this way. Will do, as the higher-ups say. :slightly_smiling_face:

File: cmd/nuntium/mediator.go
Comment (jezek) [inline cmd/nuntium/mediator.go]: Now I know the code is working, the last log could be commented out. Just leave the logs after error.

**Comment by dobey** [comment]
> Yep, it would be more logical. Maybe it's also fine this way. Will do, as the higher-ups say.

I think it would be better, or at least the first change should be to merge in the changes from the phonedations repo, by adding it as a remote and doing a `merge` or a `cherry-pick` rather than applying a diff, so that the commit history is properly preserved. The history should be compatible enough to do so, at least. Then all the imports and references to phonedations can be added separately, so there's a clear line of separation between the two.

Thanks!

**Review by dobey — CHANGES_REQUESTED** [review]
See previous comment.

**Comment by jezek** [comment]
@dobey You mean like this, or split the PR into two (mmsEnabled, imports)?

**Comment by dobey** [comment]
> You mean like this, or split the PR into two (mmsEnabled, imports)?

Two separate PRs would probably be better. One for merging the "upstream" changes in (like maybe adding the remote and creating a new branch based on its master, and just PR that into xenial here), and a second with your changes. Or maybe three PRs since the "error logging and refactor mmsEnabled" bit is also separate from the replacing ubuntu-phonedations bit.

Thanks!

File: README.md
Comment (Flohack74) [inline README.md]: Its _continuation_

**Comment by jezek** [comment]
@dobey 
> Two separate PRs would probably be better. One for merging the "upstream" changes in (like maybe adding the remote and creating a new branch based on its master, and just PR that into xenial here), and a second with your changes. Or maybe three PRs since the "error logging and refactor mmsEnabled" bit is also separate from the replacing ubuntu-phonedations bit.

To be sure, we are on the same page.

I've added upstream (ubuntu-phonedations/master) and then (after resolving some conflicts, due to new merged code in xenial) merged with xenial (see https://github.com/jezek/nuntium/tree/xenial_-_merge-upstream). This would be the PR no 1. 

Then the second would be error logging and refactoring of mms-enabled logic. 

An then the third would be this (without the first two commits and fixed reviews). Right?

**Comment by dobey** [comment]
> Right?

Sounds good, yeah. Main concern was to preserve the upstream history. Thanks.

File: README.md
Comment (jezek) [reply README.md]: fixed

**Comment by jezek** [comment]
@dobey Sorry for my delay. I've added PRs #6 & #7 , which should precede this one.

**Comment by jezek** [comment]
Ok, now the PRs #6 & #7 are merged to xenial branch. What other changes need to be done to accept this PR?

**Review by dobey — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**
